frontend/src/components/common/Filter/common/style.ts

```diff
@@ -35,7 +35,6 @@ export const SelectionIcon = styled(Icon)`
       width: 14px;
     }
   }
-}
```

Comment:
woah we've had this invalid closing bracket for a while lol 😆 Thanks for fixing this!